src/js/components/Tree/Tree.js

```diff
@@ -0,0 +1,161 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { ThemeContext } from 'styled-components';
+import { FormDown } from 'grommet-icons/icons/FormDown';
+import { FormNext } from 'grommet-icons/icons/FormNext';
+import { FormPrevious } from 'grommet-icons/icons/FormPrevious';
+
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { List } from '../List';
+import { Text } from '../Text';
+
+const Tree = ({ children, mode, data }) => {
+  const theme = useContext(ThemeContext);
+
+  const [depth, setDepth] = useState(0);
+  // an array of the path names
+  const [breadcrumbs, setBreadcrumbs] = useState([]);
```

Comment:
A minor idea: rename "breadcrumbs" to "ancestors"? Technically, breadcrumbs mean "how I got here". In the case of a deep link, the user may not have navigated sequentially through the tree.

Actually, based on comments further down in this file, perhaps "path" would be a more precise name?

Reply:
Changed to `path`.